addons/crm_livechat/data/crm_livechat_chatbot_data.xml

```diff
@@ -9,6 +9,7 @@
 
     <record id="chatbot_script_lead_generation_bot" model="chatbot.script">
         <field name="title">Lead Generation Bot</field>
+        <field name="operator_id" ref="base.user_root"/>
```

Comment:
note: new fields/values should use "Agent" name rather than "Operator" (we'll eventually do a PR to rename the remaining one)

Reply:
Not sure yet but I wonder if we wouldn't want to call it `agent_user_id` to be more clear about what it contains.